core/src/test/java/org/infinispan/jmx/TxInterceptorMBeanTest.java

```diff
@@ -106,8 +106,8 @@ public void testRemoteRollback() throws Exception {
 
    private void assertCommitRollback(int commit, int rollback, ObjectName objectName) throws Exception {
       String commitCount = threadMBeanServer.getAttribute(objectName, "Commits").toString();
-      assert Integer.valueOf(commitCount) == commit : "expecting " + commit + " commits, received " + commitCount;
```

Comment:
May as well convert to `AssertJunit#assertEquals`

Reply:
Yes, I usually do that conversion in test methods that are affected by the PR. If there just a few places in that class I can do them in this PR.

Reply:
done